packages/visualizer-browser/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{
+  "compilerOptions": {
```

Comment:
extend the root `tsconfig.options.json` and then remove options here that are dupes, but keep ones that are different

Reply:
I added the `extends` key, but otherwise `tsconfig.options.json` is mostly empty so I was not sure what to remove from this `tsconfig.json`.

Reply:
The `tsconfig.options.json` extends from another config, so you'll need to inspect that one.